.github/workflows/deploy.yml

```
@@ -14,55 +15,30 @@ jobs:
     - name: Checkout source
       uses: actions/checkout@v3
 
-    - name: EC2에 AI Server 디렉터리 생성
-      uses: appleboy/ssh-action@v0.1.7
+    - name: 도커 허브 로그인
+      uses: docker/login-action@v2
```

actionlint (1.7.7)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)